src/utilities/make_state_vector_file.py

```diff
@@ -485,21 +500,23 @@ def make_country_state_vector_file(config_path, land_cover_pth, hemco_diag_pth,
     save_pth = sys.argv[4]
     # If the Region of Interest is specified, then the make_country_state_vector_file function is called. Else, the make_rectilinear_state_vector_file function is called
     try:
-        if config["RegionOfInterest"] is not None:
+        if (config["RegionOfInterest"] is not None) and config["CreateAutomaticRectilinearStateVectorFile"] == False:
             make_country_state_vector_file(
             config_path, 
             land_cover_pth, 
             hemco_diag_pth, 
             save_pth
             )
-        else:
-            print("No specified ROI")
+        elif (config["RegionOfInterest"] is None) and config["CreateAutomaticRectilinearStateVectorFile"] == True:
+            print("Creating Rectilinear StateVector File")
             make_rectilinear_state_vector_file(
             config_path, 
             land_cover_pth, 
             hemco_diag_pth, 
             save_pth,
             )
+        else:
+            print("ERROR: The method of creating a statevector was not specified correctly.")
```

Comment:
I think you should [raise](https://www.w3schools.com/python/ref_keyword_raise.asp) an error here rather than just a print statement.